package.json

```diff
@@ -49,6 +52,7 @@
     "prepublish": "npm run compile",
     "lint": "eslint .",
     "test": "run-s lint format:check",
-    "version": "npm run babelify && git add browser/gotrue.js"
+    "version": "npm run babelify && git add browser/gotrue.js",
+    "release": "CI=true yarn install && yarn run version && semantic-release && npm publish"
```

Comment:
Prefixing environment variables does not work in `cmd.exe`, if anyone on Windows was to work in this repository.
Should we use [`cross-env`](https://github.com/kentcdodds/cross-env)?

Reply:
Just saw this comment, I'll add it.